src/quota_aggregator_impl.cc

```diff
@@ -258,20 +237,47 @@ ::google::protobuf::util::Status QuotaAggregatorImpl::FlushAll() {
 
 // OnCacheEntryDelete will be called behind the cache_mutex_
 // no need to consider locking at this point
+// This is called when an cached item is older than refresh_interval
+// as SetAgeBasedEviction() is called at the constuctor.
 void QuotaAggregatorImpl::OnCacheEntryDelete(CacheElem* elem) {
```

Comment:
I guess I partially understand this PR. It would be better if you document the latest cache behavior in one pager, like the cache flow diagram in the original design. It would be helpful for code reviewing and future maintaining. 

https://docs.google.com/document/d/1QPk2wPyJOVARlOlZWQSjfVfeL08piRfdT9UE7OKkrk8/edit#bookmark=id.sihqnkjolzhm

Reply:
Thanks.  

Updated the doc with following

For a cached negative response, a CHECK_ONLY call with 1 request cost will be sent to the server at the end of the refresh interval to detect quota availability and to switch the response from negative to positive earlier. Otherwise the negative cached item will reject the next request even the quota is available on the server.

In this section. "High-level algorithm"

Reply:
Also added a link in the header file comment to the design doc for future reviewers.